article/_posts/2022-04-14-MultiModalVersatileNetworks.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: review
+title: "Self-Supervised MultiModal Versatile Networks"
+tags: deep-learning representation-learning self-supervised multi-task
+author: "Nathan Painchaud"
+cite:
+    authors: "Jean-Baptiste Alayrac, Adria Recasens, Rosalia Schneider, Relja Arandjelović, Jason Ramapuram, Jeffrey De Fauw, Lucas Smaira, Sander Dieleman, Andrew Zisserman"
+    title:   "Self-Supervised MultiModal Versatile Networks"
+    venue:   "NeurIPS 2020"
+pdf: "https://proceedings.neurips.cc/paper/2020/file/0060ef47b12160b9198302ebdb144dcf-Paper.pdf"
+---
+
+
+# Highlights
+- Introduce the notion of a MultiModal Versatile (MMV) network, that can ingest multiple modalities and outputs
+representations useful for downstream tasks in multiple modalities;
+    - The papers especially studies how best to combine the modalities, so that representations respect four properties
```

Comment:
- "The papers especially studies": "The paper especially studies"
- Is the different indentation intended?

Reply:
Which are these "four properties"?

Reply:
The four properties are described in the introduction, but I'll rephrase this not to refer to specific concepts not yet described.

Reply:
Done in c9d2a78.